### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by duanyytop**
update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,49 @@
 
 # Neuron Wallet (Android)
 
-Welcome to Neuron's open source Android app! 
+Overview
+===============
+
+Neuron is an open source blockchain wallet which supports Ethereum and [AppChain](https://docs.nervos.org/#/). It supports almost tokens of Ethereum and [AppChain](https://docs.nervos.org/#/), such as ETH, ERC20, ERC721, and also supports all kinds of DApps of Ethereum and [AppChain](https://docs.nervos.org/#/) , such as cryptokitties, Fomo3D, 0xproject and so on.
```
Comment (ashchan) [inline README.md]: Should this

'It supports almost tokens'

be

`It supports most tokens` instead?

Comment (ashchan) [reply README.md]: `such as` and `and so on` are redundant.

File: README.md [hunk 2]
```diff
@@ -2,19 +2,49 @@
...
+
+Usage
+===============
+
+### Private key and address
+
+Neuron is a blockchain wallet which supports Ethereum and [AppChain](https://docs.nervos.org/#/) (AppChain uses same SECP256k1 signature algorithm), you can use same private key and address. **Neuron nerver save your private key directly, you need input password to signature every transaction. If you forget your private key, Neuron can not find and recover it, so you should save private key (keystore and mnemonic) carefully.**
```
Comment (ashchan) [inline README.md]: * `never saves`
* `you need to`
* `input password to sign every transaction`

File: README.md [hunk 3]
```diff
@@ -2,19 +2,49 @@
...
+
+[AppChain](https://docs.nervos.org/#/) is a blockchain solution which includes blockchain kernel CITA, Neuron wallet, blockchain browser [Microscope](https://github.com/cryptape/microscope), cache server [ReBirth](https://github.com/cryptape/re-birth) and SDKs. [AppChain](https://docs.nervos.org/#/) supports Ethereum solidity contract, so all ERC contracts can run on AppChian directly. AppChain is an open source blockchain solution, so you can publish your blockchain coin by yourself and set any name you like. All tokens on AppChian can display in Neuron.
+
+### DApp
+
+Neuron is also a DApp browser, which supprts DApps of Ethereum and [AppChain](https://docs.nervos.org/#/). Almost popular Ethereum DApps, such as cryptokitties, Fomo3D, 0xproject and so on, can run in Neuron directly. Neuron also supprts AppChain DApps, which can be easily migrated from Ethereum. You can get more information about [how to develop an AppChain DApp](https://docs.nervos.org/nervos-appchain-docs/#/quick-start/build-dapp).
```
Comment (ashchan) [inline README.md]: `Almost all`.

`such as` and `and so on` like my previous comment.

File: README.md (hunk 1 above)
Comment (duanyytop) [reply README.md]: OK

File: README.md (hunk 2 above)
Comment (duanyytop) [reply README.md]: OK

File: README.md (hunk 3 above)
Comment (duanyytop) [reply README.md]: OK

**Pull request merged**